### PR TITLE
Fixed labels order

### PR DESCRIPTION
**Pull request by ActiveChooN**
### Motivation and context

External bug report

### How has this been tested?

No tests, due to no issue reproduction on development setup.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
~~- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/engine/serializers.py [hunk 1]
```diff
@@ -79,6 +79,7 @@ class LabelSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Label
         fields = ('id', 'name', 'color', 'attributes', 'deleted')
+        ordering = ['-id']
```
Comment (nmanovic) [inline cvat/apps/engine/serializers.py:82]: Could you please add a link on documentation about the field? I can see such parameter for Django Models: https://docs.djangoproject.com/en/3.2/ref/models/options/#ordering

Comment (nmanovic) [reply cvat/apps/engine/serializers.py:82]: Or may be on the DRF source code. Because in IAM PR I deleted all such fields.

Comment (ActiveChooN) [reply cvat/apps/engine/serializers.py]: Could not find any reference. So, moved to custom prefetching of related fields.

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed Interaction handler keyboard handlers (<https://github.com/openvinotoolkit/cvat/pull/3881>)
 - Points of invisible shapes are visible in autobordering (<https://github.com/openvinotoolkit/cvat/pull/3931>)
 - Order of the label attributes in the object item details(<https://github.com/openvinotoolkit/cvat/pull/3945>)
+- Order of label in tasks and projects (<https://github.com/openvinotoolkit/cvat/pull/3987>)
```
Comment (nmanovic) [inline CHANGELOG.md:36]:
```suggestion
- Order of labels in tasks and projects (<https://github.com/openvinotoolkit/cvat/pull/3987>)
```

**Review by nmanovic — APPROVED** [review]

**Pull request merged**